bessctl/test_sugar.py

```diff
@@ -0,0 +1,100 @@
+import fnmatch
+import os
+import sugar
+import unittest
+
+
+class TestSugar(unittest.TestCase):
+    """
+    All scripts in conf/ will be dynamically added here as individual
```

Comment:
Seems both C++ and Python are working now: https://codecov.io/gh/sangjinhan/bess/branch/travis_test8